packages/vuepress-theme-odyssey/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@okta/vuepress-theme-odyssey",
+  "version": "0.6.0",
+  "description": "VuePress theme for Odyssey Documentation",
+  "author": "Okta, Inc.",
+  "license": "Apache-2.0",
+  "private": true,
+  "main": "index.js",
+  "devDependencies": {
+    "html-loader": "^1.1.0",
+    "node-sass": "4.13.1",
+    "sass-loader": "8.0.2",
+    "stylelint": "^9.3.0",
+    "stylelint-checkstyle-formatter": "^0.1.1",
+    "stylelint-config-recommended": "^2.1.0",
+    "stylelint-config-sass-guidelines": "^5.0.0",
+    "stylelint-config-standard": "^18.2.0",
+    "stylelint-order": "^0.8.1",
+    "stylelint-scss": "^3.2.0"
+  },
+  "dependencies": {
+    "@okta/odyssey": "^0.6.0",
+    "@okta/odyssey-icons": "^0.6.0",
+    "focus-trap": "^6.1.3",
+    "focus-trap-vue": "https://github.com/arnoldsandoval-okta/focus-trap-vue.git#with-build",
```

Comment:
Move it within `okta/` prefix?

Reply:
This is a temporary fork, I have contributed the code back to the repository and it has been accepted. We will use the actual package once my changes are released -- I am keeping an eye on this!

Reply:
Mentioned [here](https://github.com/okta/odyssey/pull/728/files/b5af62ef66aa4436302124ef087b2bc69d216b14#r510377049), but let's add a JIRA/GitHub issue to track this so we don't forget.

Reply:
#744 has been logged